docs/organization/integrations/feature-flag/launchdarkly/index.mdx

```diff
@@ -4,6 +4,8 @@ sidebar_order: 1
 description: Learn about Sentry's LaunchDarkly integration. LaunchDarkly enables organizations to use Sentry errors as a metric in their LaunchDarkly experiments.
```

Comment:
we should probably update this description to be more generic -- right now it's about metrics, but we're adding FF info to this page too!